### PR TITLE
Add example using glamorous 💄

### PR DESCRIPTION
**Pull request by paulmolluzzo**
I created a quick example using [glamorous](https://github.com/kentcdodds/glamorous). It's very similar to `glamor` (it depends on it), but why not. :man_shrugging:

### CODE REVIEW

**Comment by paulmolluzzo** [comment]
[Now](https://with-glamorous-ogeqmhyius.now.sh/).

There's a big FOUC. I'm not sure if it's my example or `glamorous`. 😟

**Comment by kentcdodds** [comment]
That FOUC is super sad. I would be spelled if it were glamorous. But I suppose it could be. Does the Glamor example do that?

**Comment by paulmolluzzo** [comment]
@kentcdodds Now: https://with-glamor-jvryhwqqfq.now.sh

---
@rauchg it would be great if all the examples were hosted on now.sh already.

**Comment by kentcdodds** [comment]
Ah, no FOUC on that because it's doing server-side rendering with rehydration. You can actually do this same thing with glamorous! It's actually the exact same process (using `glamor` APIs). Can you see if that will solve the FOUC issues?

**Comment by paulmolluzzo** [comment]
[Fixed the FOUC](https://with-glamorous-zrqwerosse.now.sh).

**Comment by kentcdodds** [comment]
This is excellent! Thank you very much :)

File: examples/with-glamorous/pages/index.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import React from 'react'
-import { css } from 'glamor'
+import { style, rehydrate, css } from 'glamor'
 import glamorous from 'glamorous'
 
+// Adds server generated styles to glamor cache.
```
Comment (ads1018) [inline examples/with-glamorous/pages/index.js]: Is this required on every page? Can we make this more DRY?

Comment (paulmolluzzo) [inline examples/with-glamorous/pages/index.js]: You could probably put this in a global component like a layout and get away with it. I haven't given it a try though.

FWIW, I'd call this a "working example" more than a "best practices example". 😉

Comment (ads1018) [inline examples/with-glamorous/pages/index.js]: Cool. Out of curiosity what are we rehydrating here? What's stored insides `ids`?

Comment (paulmolluzzo) [inline examples/with-glamorous/pages/index.js]: We're rehydrating the CSS by shoving the computed styles into the DOM in the `head`. This is actually being done by `glamor`. See here for more: https://github.com/threepointone/glamor/blob/master/docs/server.md

The `ids` are the hashes that are part of the generated classes uses in the CSS.

Here's an example I just pulled from a different project:
```js
__NEXT_DATA__ = {"props":{},"pathname":"/","query":{},"buildId":"-","buildStats":null,"assetPrefix":"","err":null,"ids":["14e33nr","151tlpa","63oe3q","35577l","wfyxwh","858wo0","2y5bqi","1nvbmsg","17186zx","cuyszr","ehn2ke","6l3qzw","1jfbwxw","gzrewh","1jh79q9","kv69lr","ubp1jq"]}
```

Comment (paulmolluzzo) [inline examples/with-glamorous/pages/index.js]: @ads1018 You can DRY this up by including in a component (like a layout component) and using that on all the pages. See here: https://github.com/kentcdodds/glamorous-website/pull/14

Comment (ads1018) [inline examples/with-glamorous/pages/index.js]: Ah perfect. This is exactly what I was thinking. Thanks!

**Pull request merged**